docs/gateway/nas.md

```diff
@@ -49,6 +49,36 @@ mc ls mynas
 [2017-02-26 22:10:11 PST]     0B test-bucket1/
 ```
 
+## Breaking changes
+
+There will be a breaking change after the release version 'RELEASE.2020-06-18T02-23-35Z'.
```

Comment:
RELEASE.2020-06-22T03-12-50Z its now this version :-)

Reply:
Changed it. Thanks!